src/CtaLink/CtaLink.js

```diff
@@ -0,0 +1,268 @@
+        break;
+
+      case 'outline':
+        levers.variant = 'su-bg-transparent hocus:su-bg-transparent su-text-digital-red hocus:su-text-black su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black';
+        break;
+
+      case 'ghost':
+        levers.variant = 'su-bg-transparent su-text-white hocus:su-text-white su-border-2 su-border-white su-border-solid';
+        break;
+    }
+  }
+
+  // size
+  if (size && ctaLinkButtonSizes.includes(size) && isButton) {
+    switch (size) {
+      case 'default':
+        levers.size = 'su-px-26 su-pt-10 su-pb-11 su-text-16 md:su-text-20';
+        break;
+
+      case 'big':
+        levers.size = 'su-px-34 su-py-15 su-text-20 md:su-text-24';
+        break;
+
+      case 'small':
+        levers.size = 'su-px-19 su-py-9 su-text-16 md:su-text-18';
+        break;
+    }
+  }
+
+  // icon
+  let heroicon = '';
+
+  if (icon && ctaLinkIcons.includes(icon)) {
+    switch (icon) {
+      case 'action':
+        heroicon = 'chevron-right';
+        levers.icon = 'su-h-1em su-w-1em su-ml-4 su--mt-2';
+        break;
+
+      case 'more':
+        heroicon = 'arrow-narrow-right';
+        levers.icon = 'su-h-09em su-w-09em su-ml-5 su--mt-2';
+        break;
+
+      case 'external':
+        heroicon = 'arrow-up';
+        levers.icon = 'su-h-08em su-w-08em su-ml-4 su--mt-2 su-transform su-rotate-45 group-hocus:su-rotate-45';
+        break;
+
+      case 'lock':
+        heroicon = 'lock-closed';
+        levers.icon = 'su-h-08em su-w-08em su-ml-4 su--mt-3';
+        break;
+
+      case 'download':
+        heroicon = 'download';
+        levers.icon = 'su-h-08em su-w-08em su-ml-4 su--mt-3';
+        break;
+
+      case 'video':
+        heroicon = 'play';
+        levers.icon = 'su-h-08em su-w-08em su-ml-6 su--mt-3';
+        break;
+
+      case 'email':
+        heroicon = 'mail';
+        levers.icon = 'su-h-08em su-w-08em su-ml-7 su--mt-2';
+        break;
+
+      case 'jump':
+        heroicon = 'chevron-down';
+        levers.icon = 'su-h-1em su-w-1em su-ml-4 su--mt-2';
+        break;
+
+      case 'none':
+        break;
+    }
+  }
+
+  // animate
+
+  // Common classes if animation is needed
+  if ((animate && ctaLinkAnimations.includes(animate)) && animate !== 'none') {
+    levers.animate = 'su-transition-transform group-hocus:su-transform ';
+  }
+
+  // Add specific classes for each type of animation
+  if (animate && ctaLinkAnimations.includes(animate)) {
+    switch(animate) {
+      case 'right':
+        levers.animate += 'group-hocus:su-translate-x-02em';
+        break;
+
+      case 'top-right':
+        levers.animate += 'group-hocus:su-translate-x-01em group-hocus:su--translate-y-01em';
+        break;
+
+      case 'down':
+        levers.animate += 'group-hocus:su-translate-y-02em';
+        break;
+
+      case 'none':
+        break;
```

Comment:
This is not necessary and can be removed.

Reply:
It would work if I remove this - it is not needed at all, just wondered if having this would make it clearer (so will have all the options in the lever map to its own case) so more like a documentation thing. If you think it's better not to have it, I'll be happy to remove.

Reply:
This has been removed.